ui/src/style/fonts/icon-font.scss

```diff
@@ -0,0 +1,140 @@
+  &.dash-e:before {content: "\e926";}
+  &.dash-f:before {content: "\e927";}
+  &.dash-g:before {content: "\e928";}
+  &.dash-h:before {content: "\e929";}
+  &.dash-i:before {content: "\e92a";}
+  &.dash-j:before {content: "\e92b";}
+  &.access-key:before {content: "\e921";}
+  &.crown2:before {content: "\e94b";}
+  &.crown:before {content: "\e90f";}
+  &.server2:before {content: "\e94c";}
+  &.server:before {content: "\e914";}
+  &.cpu:before {content: "\e915";}
+  &.cubo-node:before {content: "\e919";}
+  &.cubo-uniform:before {content: "\e91a";}
+  &.user-outline:before {content: "\e91c";}
+  &.refresh:before {content: "\e949";}
+  &.pause:before {content: "\e94a";}
+  &.clock:before {content: "\e91b";}
+  &.triangle:before {content: "\e92c";}
+  &.square:before {content: "\e93e";}
+  &.diamond:before {content: "\e93f";}
+  &.octagon:before {content: "\e92d";}
+  &.circle:before {content: "\e940";}
+  &.cube:before {content: "\e92e";}
+  &.component-e:before {content: "\e92f";}
+  &.component-d:before {content: "\e930";}
+  &.component-c:before {content: "\e931";}
+  &.component-b:before {content: "\e932";}
+  &.component-a:before {content: "\e933";}
+  &.pulse-a:before {content: "\e934";}
+  &.pulse-b:before {content: "\e935";}
+  &.pulse-c:before {content: "\e936";}
+  &.pulse-d:before {content: "\e937";}
+  &.pulse-e:before {content: "\e938";}
+  &.brush:before {content: "\e939";}
+  &.keynote:before {content: "\e93a";}
+  &.polaroid:before {content: "\e93b";}
+  &.alpha-a:before {content: "\e93c";}
+  &.alpha-b:before {content: "\e93d";}
+  &.export:before {content: "\e941";}
+  &.expand-b:before {content: "\e942";}
+  &.expand-c:before {content: "\e943";}
+  &.expand-a:before {content: "\e944";}
+  &.undo:before {content: "\e945";}
+  &.redo:before {content: "\e946";}
+  &.heroku:before {content: "\e947";}
+  &.heroku-simple:before {content: "\e948";}
+  &.oauth:before {content: "\e94f";}
+  &.authzero:before {content: "\e951";}
+  &.shuffle:before {content: "\e94e";}
+  &.disks:before {content: "\e950";}
+  &.alert-triangle:before {content: "\f02d";}
+  &.comment:before {content: "\f02b";}
+  &.comment-discussion:before {content: "\f04f";}
+  &.heart:before {content: "\2665";}
+  &.link:before {content: "\f05c";}
+  &.location:before {content: "\f060";}
+  &.octoface:before {content: "\f008";}
+  &.pencil:before {content: "\f058";}
+  &.star:before {content: "\f02a";}
+  &.stop:before {content: "\f08f";}
+  &.sync:before {content: "\f087";}
+  &.zap:before {content: "\26a1";}
+  &.google:before {content: "\ea88";}
+  &.google3:before {content: "\ea8a";}
+  &.google-plus:before {content: "\ea8b";}
+  &.facebook:before {content: "\ea90";}
+  &.facebook2:before {content: "\ea91";}
+  &.twitter:before {content: "\ea96";}
+  &.github:before {content: "\eab0";}
+  &.trello:before {content: "\eab3";}
+  &.linkedin:before {content: "\eac9";}
+  &.linkedin2:before {content: "\eaca";}
+  &.stackoverflow:before {content: "\ead0";}
```

Comment:
Do we need these 3rd party brand icons?

Reply:
Arguably no. They're vestigial from when the theme was used on the website